plugins/package-managers/python/src/main/kotlin/utils/PythonInspectorExtensions.kt

```diff
@@ -23,6 +23,7 @@ import java.io.File
 
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.model.utils.toPurl
```

Comment:
I think detekt expects this import below `import org.ossreviewtoolkit.model.VcsType`. You could also run `./gradlew detekt --auto-correct` to automatically fix the import order.

Reply:
Done